scripts/cibuild

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function usage() {
+    echo -n \
+         "Usage: $(basename "$0")
+Build application for staging or a release.
+"
+}
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+      if [ -n "${RF_DOCS_BUCKET}" ]; then
+          echo "Building docs container"
+          docker-compose build docs
+
+          echo "Building dependencies"
+          docker-compose run --rm docs install
+
+          # echo "Running tests"
+          # "${DIR}/test"
+          # echo "All tests pass!"
+          echo "Building static bundle"
+          if [ "${1:-}" = "--prod" ]; then
```

Comment:
Looking at the documentation, I think that we want all builds that come out of this process to use `--prod`.